### ISSUE TITLE
🐛 Bug: Unable to change python versions

### ISSUE THREAD

**Issue by pittmanriley** [ersilia-os/ersilia#708]
### Describe the bug.

I'm unable to change the version of my Python system to 3.10. When I type in the command python --version, it shows me 3.8.9, but when I type python3 --version, it shows me 3.10.0. We believe this is causing me to have issues with getting the model hub to work. 

### Describe the steps to reproduce the behavior

1. `conda activate ersilia` 
2. `python --version` 
- this will get me 3.8.9
3. `python3 --version` 
- this will get me 3.10.0
4. When inside the conda environment, run`ersilia catalog`
- this gets an error saying 'ModuleNotFoundError: No module named 'bentoml.cli'

### Expected behavior.

I expect my conda environment to use the python version 3.10, and for the ersilia catalog command to work functionally. 

### Screenshots.

<img width="997" alt="Screen Shot 2023-06-19 at 12 07 10 PM" src="https://github.com/ersilia-os/ersilia/assets/94929717/bacf83fd-666a-4929-8f63-c7c4f2374076">

### Operating environment

macOS version 12.3.1, and I have an M1 chip

**Comment by miquelduranfrigola** [comment]
Thanks @pittmanriley this is most likely a python paths / conda paths issue rather than an Ersilia issue.

Inside the conda environment, could you try to install `ersilia` in development mode in both python paths?

```
cd ersilia
python -m pip install -e ersilia
python3 -m pip install -e ersilia
```

This is not an ideal solution, but let see if it gives us some hints.

**Comment by GemmaTuron** [comment]
Hi @pittmanriley 
Can you please update this issue so that we can close it?

**Comment by pittmanriley** [comment]
Hi @GemmaTuron

Yes, this issue is resolved. There ended up being two issues here. First, the BentoML issue was solved by installing the correct version that was compatible with Ersilia by using the following command within the ersilia conda environment: `python -m pip install -U git+https://github.com/ersilia-os/bentoml-ersilia.git`

The second issue wasn't as big of an issue, but I noticed that when I ran `which python` in the terminal, it told me this: `python: aliased to /usr/bin/python3`. So, I removed this alias statement from my .zshrc file and now both my python3 and python versions are the same.
